mappings/net/minecraft/entity/LightningEntity.mapping

```diff
@@ -1 +1,16 @@
 CLASS app net/minecraft/entity/LightningEntity
+	FIELD b randomSeed J
+	FIELD c ticksBeforeDespawn I
+	FIELD d fireSpawnCount I
+	FIELD e preventDamage Z
```

Comment:
`harmless`, maybe? Negative connotation for booleans is bad.

Reply:
"noDamage"?